_docs/style/spacing.md

```diff
@@ -0,0 +1,290 @@
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption: "Table leaf-node spacing"][class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/spacing/spacing-system-leaf-node-margins.png){: width="566px"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### List items
+
+A list is a structure made of multiple items of homogenous data. It is important that the list items are not spaced apart like paragraphs, which typically hold heterogenous ideas.
+
+To improve legibility, list items still need to be spaced apart. Many spacing values were considered before landing on four pixels.  
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption: "List leaf-node spacing"][class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/spacing/spacing-system-lists.png){: width="531px"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Full-width items in a bound container
+
+A modification is made for list items that extend to the left and right edge of a visually bound container. The example is items in a drop-down menu. 
+
+In cases like this the container does not receive a square inset.  To move the first item away from the top edge of the container, the first item gets a top space of four pixels.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption: "Full-width items in a bound container"][class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/spacing/spacing-system-full-width-items-in-a-bound-container.png){: width="536px"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Form elements
+
+We found that eight pixels of vertical space between consecutive, stacked inputs works best.  Eight pixels of space also applies to consecutive, horizontally inline inputs.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption: "Form element leaf-node spacing"][class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/spacing/spacing-system-forms.png){: width="462px"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Paragraphs
+
+Paragraph line-height of twenty-four pixels was derived from visual explorations and validated using WCAG SC 1.4.8 guideline, which recommends a line-height of at least 1.5 within paragraphs. Spacing beloew paragraphs is sixteen pixels.
```

Comment:
s/guideline/guidelines

Also: Possibly add a link to the guidelines if this would benefit the text.